shared/api/system.go

```diff
@@ -7,7 +7,7 @@ type SystemConfig struct {
 
 type ConfigNetwork struct {
 	Address        string `json:"address,omitempty"         yaml:"address,omitempty"`
-	Port           int    `json:"port,omitempty"            yaml:"port,omitempty"`
+	Port           string `json:"port,omitempty"            yaml:"port,omitempty"`
```

Comment:
I'd probably expect the API type to be an integer. The zero value would then use the default port.

Reply:
I thought we discussed above to not have any default port, and not start a listener at all until an address/port is set explicitly?

Reply:
Or is it that we just don't want to use wildcard addresses, so require an address to be set explicitly but the port can have a default?

Reply:
Yeah, we shouldn't start a listener until an address has been configured.

So basically:
 - Address="" => Don't start listener (regardless of Port value)
 - Address="::" => Listens on everything, uses default port if Port is set to 0
 - Address="0.0.0.0" => Listen on all IPv4 addresses